pkg/sdk/go/channels_test.go

```diff
@@ -15,7 +15,10 @@ import (
 )
 
 var (
-	channel      = sdk.Channel{ID: "001", Name: "test"}
+	ch1ExtID     = sdk.Channel{ID: "fe6b4e92-cc98-425e-b0aa-000000000001", Name: "test1"}
+	ch2ExtID     = sdk.Channel{ID: "fe6b4e92-cc98-425e-b0aa-000000000002", Name: "test2"}
+	channel1     = sdk.Channel{Name: "test1"}
+	Prefix       = "fe6b4e92-cc98-425e-b0aa-"
```

Comment:
Do you need to export this? `chPrefix` would be better in any case

Reply:
moving to chPrefix

Reply:
btw, you could also use this prefix here:
```
 	ch1ExtID     = sdk.Channel{ID: chPrefix +-"000000000001", Name: "test1"} 
 ```